docs/sdks_03_safe_apps.md

```diff
@@ -29,6 +44,22 @@ We provide Safe App developers with reusable react components to make it easy to
 [UI Guidelines](assets/safe_apps_guides.png)
 
 
+### Existing Safe apps
+
+This is a list of known public repositories containing Safe apps for reference:
+
+- [Open Zeppelin](https://github.com/OpenZeppelin/upgrades-safe-app)
+- [Balancer - Pooling](https://github.com/TomAFrench/pool-management/tree/safe-app)
+- [Balancer - Exchange](https://github.com/balancer-labs/balancer-exchange)
+- [1inch Exchange](https://github.com/CryptoManiacsZone/gnosis.1inch.exchange)
+- [Synthetix](https://github.com/protofire/safe-app-synthetix-mintr)
+- [Sablier](https://github.com/TomAFrench/sablier-safe-app)
+- [Compound](https://github.com/gnosis/safe-react-apps/tree/development/src/apps/Compound)
+- [Transaction builder](https://github.com/gnosis/safe-react-apps/tree/development/src/apps/TxBuilder)
+- [Aave](https://github.com/mrdavey/gnosis-safe-app)
+- [Idle](https://github.com/krzysu/safe-app-idle)
```

Comment:
Should we link the list from the other pr?

Reply:
Here we link the code whereas in the other lists only the projects, so would keep separate.